src/openms/source/FORMAT/DATAACCESS/CsiFingerIdMzTabWriter.cpp

```diff
@@ -89,7 +89,6 @@ void CsiFingerIdMzTabWriter::read(const std::vector<String> & sirius_output_path
         // results from scan were not assigned to a feautre
         if (feature_id == "id_0") {feature_id = "null";}
```

Comment:
would not change the value of `feature_id`. Perhaps only use the "null" value where `feature_id` is read, otherwise I'd use a new variable name.

Reply:
Right now, feature_id is processed from name of the .csv file.
If no feature_id was associated the regex will still pick up an id_0. 
The writer will then write a "null" in the feautre_id column. 
In the case a feature_id is present it should look like:
id_3456783223456734

Reply:
I think Lukas means that it is confusing/hard to follow if the feature_id is overwritten.
I think what he means is to better set the id to e.g. "UNKNOWN_ID" (ideally a constant defined as static constexpr) instead of id_0.
When writing to mzTab you would check if it is  == this value and if so write out "null".